hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/reconfig/ReconfigureCommands.java

```diff
@@ -53,10 +59,20 @@ public class ReconfigureCommands implements Callable<Void>,
   private CommandSpec spec;
 
   @CommandLine.Option(names = {"--address"},
-      description = "node address: <ip:port> or <hostname:port>",
+      description = "node address: <ip:port> or <hostname:port>. For Datanode" +
+          " the parameter '" + BULK_OPERATION_IDENTIFIER + "' is supported. " +
+          "This will send reconfiguration requests to all available DataNodes" +
+          " in the IN_SERVICE operational state.",
       required = true)
+
   private String address;
 
+  @CommandLine.Option(names = {"--type", "-t"},
+      description = "Specifies the type of the server. Currently, only " +
+          "'datanode' is supported.",
+      required = false)
+  private String type;
```

Comment:
Instead of requiring a fake address, I would prefer a dedicated flag, something like:

```
ozone admin reconfig --in-service-datanodes start
```

Reply:
OK,  in fact, the format of this command is to simulate the command format of HDFS. but the command of `ozone admin reconfig --in-service-datanodes start` looks like more friendly. 
I will update this.

Reply:
done.